books/tests/sample_epubs.py

```diff
@@ -15,10 +15,14 @@
 # calculated automatically based on 'filename'.
 EPUBS_ALL = [SampleEpub('epub30-spec', 'epub30-spec.epub',
                         os.path.join(RSRC_DIR, 'epub30-spec.epub'),
-                        True, False),
+                        True, True),
              SampleEpub('figure-gallery', 'figure-gallery-bindings.epub',
                         os.path.join(RSRC_DIR, 'figure-gallery-bindings.epub'),
                         True, True),
+             # Cover type = guide
+             SampleEpub('israelsailing', 'israelsailing.epub',
+                        os.path.join(RSRC_DIR, 'israelsailing.epub'),
+                        True, True),
```

Comment:
Nice, a new testing epub is always nice! Can you update the `README.md` with basic info about the new `israelsailing` entry?

Also, if `epub30-spec` indeed has a cover (I blindy assumed it did not, as the previous version of the parser could not find it), would you be so king of trying to find a sample epub *without* cover, and add it to the repo? This would help keeping the tests explicitly test the files without covers.